core/frontend/src/components/common/SpinningLogo.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <v-img
+    class="lds-dual-ring"
+    src="../../assets/logo.svg"
+  />
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+
+export default Vue.extend({
+  name: 'SpinningLogo',
+})
+</script>
+
+<style>
+  .lds-dual-ring {
+    display: block;
+    width: 80px;
```

Comment:
avoid `px`, use `em` to support high definition displays.

Reply:
Using `%` instead, as it's the recommended way to scale relative to parent elements. `em` are font-size dependent, so it wouldn't be the best choice here.

Reply:
`em` are not font-size dependent, they are relative to resolution, as fonts.

> Pixel is a static measurement, while percent and EM are relative measurements. The size of an EM or percent depends on its parent. If the text size of body is 16 pixels, then 150% or 1.5 EM will be 24 pixels (1.5 * 16).

Reply:
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units

Relative length units section.